examples/Cassie/run_controller_switch.cc

```diff
@@ -1,55 +1,136 @@
 #include <string>
-
 #include <gflags/gflags.h>
+#include "dairlib/lcmt_controller_switch.hpp"
+#include "dairlib/lcmt_robot_output.hpp"
+#include "systems/framework/lcm_driven_loop.h"
 #include "drake/multibody/rigid_body_tree.h"
-#include "drake/systems/analysis/simulator.h"
 #include "drake/systems/framework/diagram_builder.h"
-#include "drake/systems/lcm/lcm_interface_system.h"
 #include "drake/systems/lcm/lcm_publisher_system.h"
 
-#include "dairlib/lcmt_controller_switch.hpp"
-#include "systems/framework/controller_channel_sender.h"
-
 namespace dairlib {
 
+using drake::systems::TriggerType;
 using drake::systems::lcm::LcmPublisherSystem;
+using drake::systems::lcm::TriggerTypeSet;
 
-DEFINE_double(end_time, std::numeric_limits<double>::infinity(),
-              "End time of simulation");
-DEFINE_double(publish_rate, 1000, "Publishing frequency (Hz)");
+DEFINE_string(channel_x, "CASSIE_STATE",
+              "The name of the channel which receives state");
 DEFINE_string(controller_channel, "PD_CONTROLLER",
               "The name of the lcm channel that dispatcher_in listens to");
```

Comment:
This comment isn't very clear. This is the new channel, after the switch.

Reply:
Done